keep/api/routes/settings.py

```diff
@@ -351,23 +351,115 @@ def _print_debug(self, *args):
             super()._print_debug(*args)
 
 
-@router.get("/apikey")
-def get_api_key(
+@router.post("/apikey", description="Create API key")
+async def create_key(
+    request: Request,
+    authenticated_entity: AuthenticatedEntity = Depends(
+        AuthVerifier(["write:settings"])
+    ),
+    session: Session = Depends(get_session),
+):
+    try:
+        body = await request.json()
+        unique_api_key_id = body['name']
+    except Exception:
+        raise HTTPException(status_code=400, detail="Invalid request body")
+
+    api_key = create_api_key(
+        session=session,
+        tenant_id=authenticated_entity.tenant_id,
+        created_by=authenticated_entity.email,
+        unique_api_key_id=unique_api_key_id,
+        role=AdminRole,
+        is_system=False,
+    )
+
+    return api_key
+
+
+@router.get("/apikeys", description="Get API keys")
+def get_keys(
```

Comment:
the logic should be something like:
1. if admin - show all API keys
2. if not admin - show all API keys but get the secrets only on the keys created by this user

Reply:
fix: 
- b95d6a54d5e8ab2a6354a1b61b15dcca87c4f7dd
- 1fd33206eaf29b579d264a3f944a8d5d85841c32
- 9d9f0bba4ee5ccedb75aa9057300140b8a29f876